asm/c128/stubs_amd64.go

```diff
@@ -34,3 +34,35 @@ func AxpyInc(alpha complex128, x, y []complex128, n, incX, incY, ix, iy uintptr)
 //  	idst += incDst
 //  }
 func AxpyIncTo(dst []complex128, incDst, idst uintptr, alpha complex128, x, y []complex128, n, incX, incY, ix, iy uintptr)
+
+// DotcUnitary is
+// for i, v := range x {
```

Comment:
Note that the quoted code is indented by 2 spaces from the `//` to get `<pre>`. (throughout)